Formula/balena-cli.rb

```diff
@@ -19,11 +19,44 @@ class BalenaCli < Formula
     sha256 high_sierra: "0e670f3cc482565ddc1a166316407fab2fd008f12128a06f7f73c3bcf07a641d"
   end
 
-  depends_on "node"
+  depends_on "node@14"
+
+  on_macos do
+    depends_on "macos-term-size"
+  end
 
   def install
+    ENV.deparallelize
     system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    bin.install_symlink Dir["#{libexec}/bin/*"]
+    (bin/"balena").write_env_script libexec/"bin/balena", PATH: "#{Formula["node@14"].opt_bin}:$PATH"
+
+    # Remove incompatible pre-built binaries
+    os = OS.kernel_name.downcase
+    arch = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch.to_s
+    node_modules = libexec/"lib/node_modules/balena-cli/node_modules"
+    node_modules.glob("{ffi-napi,ref-napi}/prebuilds/*")
+                .each { |dir| dir.rmtree if dir.basename.to_s != "#{os}-#{arch}" }
```

Comment:
Right now, these don't have `darwin-arm64` prebuilds. For better performance, could locally `npm install prebuildify` and then do `npm run prebuild`, which should generate a `darwin-arm64` directory.

Otherwise, I think the slow path of non-prebuilt should work.

Reply:
I wonder if we should do something that runs `npm run prebuild` in `Language::Node` so that we can simplify always building these prebuilds from source.